src/openrct2/peep/Peep.h

```diff
@@ -774,6 +774,7 @@ struct Guest : Peep
     void UpdateEasterEggInteractions();
     void InsertNewThought(PeepThoughtType thought_type);
     void InsertNewThought(PeepThoughtType thought_type, ShopItem thought_arguments);
+    void InsertNewThought(PeepThoughtType thought_type, ride_id_t ride);
```

Comment:
```suggestion
    void InsertNewThought(PeepThoughtType thought_type, ride_id_t rideId);
```

If a variable is called ride, it should be a Ride, RCT2Ride or something like this, not just an ID, IMO.